src/tools/schema2kotlin.ts

```diff
@@ -182,6 +183,7 @@ ${imports.join('\n')}
     return `
 ${typeAliases.join(`\n`)}
 
+${this.opts.wasm ? '' : '@Generated'}
```

Comment:
I'm wondering: How about we always add the `@Generated` tag, and create an alternate import / annotation for the Wasm case? This would mean less moving parts, and give us some control of the Wasm env later on. WDYT?

Reply:
That seems like a reasonable idea.